docs/account.md

```diff
@@ -1,5 +1,219 @@
+  - `implementation` the implementation the account is using (typically `js` or `libcore`). It would directly map the account to use the corresponding [AccountBridge](./AccountBridge.md).
+  - `version` a version number we have to allow for a migration system (changing it usually will force a recalculation of operations).
+  - `currencyId` tracks the unique id of currency.
+  - `xpubOrAddress` is the "restore key" which is either a xpub or an address. basically a way (combined with some other fields) to restore the account informations.
+  - `derivationMode` will be one of the derivation modes associated to the currency. See [derivation documentation](./derivation.md).
+- `seedIdentifier` is a unique way to identify a seed the account was associated with. The important criteria is it MUST be different between 2 seeds but it SHOULD be the same within siblings accounts inside a given `derivationMode`. We usually will use, in context of BIP44, the pubKey at `purpose'/coinType'`. For other accounts that don't uses BIP44, we have simply used the account address.
+- `xpub` is the xpub if relevant. This information is redundant with the `id` and might be eventually dropped.
+- `derivationMode` identify the derivation scheme that is used. See [derivation documentation](./derivation.md). example of values: `segwit`, `unsplit`, `native_segwit`,...
+- `index` is the index of the account among a given `derivationMode`.
+- `freshAddress` is the "next" public address where a user should receive funds. In context of Bitcoin, the address will be "renewed" each time funds were received in order to allow some privacy. In other blockchains, it might never changes.
+- `freshAddressPath` represents the ["derivation path"](./derivation.md) where the `freshAddress` was taken from.
+- `freshAddresses` is an array of "future" fresh addresses. It is a generalisation of `freshAddress` and `freshAddressPath` (which will eventually deprecate them).
+- `name` is the name of the account that user have set. It is not saved or restored from any place but is defined by the user and locally saved.
+- `balance` represent the total amount of assets that this account holds.
+- `spendableBalance` represent the subset of `balance` that can be spent. Most of the time it will be equal to `balance` but this can varies in some blockchains.
+- `blockHeight` tracks the current blockchain block height.
+- `currency` is the associated crypto currency of the Account.
+- `unit` is the user defined preferred unit to view the account with. It's initialized to `currency.units[0]`.
+- `operationsCount` gives the total number of operations this account contains. This field exists because the `operations` array is not guaranteed to be complete.
+- `operations` is an array of operations sorted from the most recent to the oldest one. It might be partially containing only containing the N recent operations but can be paginated on.
```

Comment:
```suggestion
- `operations` is an array of operations sorted from the most recent to the oldest one. It might be partial, containing only the last N operations but can be paginated on.
```